feat(curriculum): Add CSS Attribute Selector Quiz

suryaanshah opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #56190 

<!-- Feel free to add any additional description of changes below this line -->

suryaanshah committed:
q1 added

suryaanshah committed:
add q2 and change styling of codeblock

suryaanshah committed:
Merge branch 'freeCodeCamp:main' into main

suryaanshah committed:
q3 added

suryaanshah committed:
q4

suryaanshah committed:
Add files via upload

suryaanshah committed:
Merge branch 'freeCodeCamp:main' into main

suryaanshah committed:
Delete curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf(1).md

suryaanshah committed:
q5

suryaanshah committed:
q6

suryaanshah committed:
q7

suryaanshah committed:
q8&9

suryaanshah committed:
q10

suryaanshah committed:
q11

suryaanshah committed:
q12

suryaanshah committed:
q13

suryaanshah committed:
q15

suryaanshah committed:
q16

suryaanshah committed:
q17

suryaanshah committed:
q18

suryaanshah committed:
q20

suryaanshah committed:
added suppose to indicate example

suryaanshah committed:
corrected option repitition

suryaanshah committed:
Update 66ed8fd0f45ce3ece4053eaf.md

ilenia-magoni commented:
@suryaanshah no, the commits are not related to this. You need to fix the title

suryaanshah commented:
Hello, does this new title fix the issue? Sorry for not making it proper the first tine

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 
----
-
-Placeholder distractor 2
```
you removed one of the distractors, please restore it

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
-Placeholder distractor 2
+```
 
 ---
+```css
+img[alt] {
+    border: 2px solid red;
+}
+```
````
````suggestion
---

```css
img[alt] {
    border: 2px solid red;
}
```
````

I'm trying my best but I am really confused by what happened to the diff of the file

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 #### --distractors--
 
-Placeholder distractor 1
+```css
+input[type="text"]:focus {
+    background-color: yellow;
+}
 
````
````suggestion
```css
input[type="text"]:focus {
    background-color: yellow;
}
````

suryaanshah replied:
do not understand what you wanted here, but I hope it is fixed now.

suryaanshah replied:
fixed this in commit in a previous commit today.

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
+```
 
-Placeholder distractor 3
+---
+```css
+input[type="text"] {
+    border: 1px solid gray;
+}
+```
````
````suggestion
---

```css
input[type="text"] {
    border: 1px solid gray;
}
```
````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
-#### --distractors--
 
-Placeholder distractor 1
+#### --distractors--
+```css
+<img src="img1.jpg" title="klematis flower" width="150" height="113">
+```
````
````suggestion
#### --distractors--

```css
<img src="img1.jpg" title="klematis flower" width="150" height="113">
```

````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 #### --answer--
-
-Placeholder answer
+```css
+<img src="img2.jpg" title="flowers" width="150" height="113">
+```
````
````suggestion

```css
<img src="img2.jpg" title="flowers" width="150" height="113">
```
````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 ---
-
-Placeholder distractor 2
+```css
+[lang="fr"] { color: blue; }
+```
````
````suggestion

```css
[lang="fr"] { color: blue; }
```
````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
-
-Placeholder question
+How would you style `<a>` elements where the title attribute starts with `"Read"`?
```
```suggestion

How would you style `<a>` elements where the title attribute starts with `"Read"`?
```

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
-
-Placeholder question
+You want to style the `<img>` elements only if their `alt` attribute does not contain the word `"thumbnail"`. Which selector would achieve this?
```
```suggestion

You want to style the `<img>` elements only if their `alt` attribute does not contain the word `"thumbnail"`. Which selector would achieve this?
```

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
-
-Placeholder question
+You need to select ordered lists (`<ol>`) that use the `type="1"` (for numerical listing). Which of the following selectors is correct?
```
````suggestion

You need to select ordered lists (`<ol>`) that use the `type="1"` (for numerical listing). Which of the following selectors is correct?
````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 1
-
+```css
+ol[type*="1"] { font-weight: bold; }
+```
````
````suggestion
```css
ol[type*="1"] { font-weight: bold; }
```

````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 2
-
+```css
+ol[type$="1"] { font-weight: bold; }
+```
````
````suggestion
```css
ol[type$="1"] { font-weight: bold; }
```

````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 3
-
+```css
+ol[type~="1"] { font-weight: bold; }
+```
````
````suggestion
```css
ol[type~="1"] { font-weight: bold; }
```

````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 #### --distractors--
 
-Placeholder distractor 1
+```css
+ul[type*="square"] { list-style-type: square; }
+```
+
````
````suggestion
```css
ul[type*="square"] { list-style-type: square; }
```
````

majestic-owl448 reviewed (commented):
first round of review, let's fix the formatting please (mainly empty lines)

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 ---
 
-Placeholder distractor 3
+```css
````
````suggestion
```html
````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,605 @@ Answer all of the questions below correctly to pass the quiz.
 ---
 
-Placeholder distractor 2
+```css
````
````suggestion
```html
````

suryaanshah commented:
Hello! I will make these suggested changes today.

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update 66ed8fd0f45ce3ece4053eaf.md

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,617 @@ Answer all of the questions below correctly to pass the quiz.
 #### --distractors--
 
-Placeholder distractor 1
+```css
+input[type="text"]:focus {
+    background-color: yellow;
+}
+
+```
````
````suggestion
```css
input[type="text"]:focus {
    background-color: yellow;
}
```
````

we can do without this empty line

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,617 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
 
-Placeholder question
+Which of the following will not run as expected with this style?
+
+```css
+<style>
+[title~="flower"] {
+  border: 5px solid yellow;
+}
+</style>
+```
+
````
````suggestion
Which of the following will not run as expected with this style?

```html
<style>
[title~="flower"] {
  border: 5px solid yellow;
}
</style>
```
````
using `style` tags, it's html

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,617 @@ Answer all of the questions below correctly to pass the quiz.
 #### --distractors--
 
-Placeholder distractor 1
+```css
````
````suggestion
```html
````

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,617 @@ Answer all of the questions below correctly to pass the quiz.
 #### --distractors--
 
-Placeholder distractor 1
+```css
+ul[type*="square"] { list-style-type: square; }
+```
+
````
````suggestion
```css
ul[type*="square"] { list-style-type: square; }
```
````

let's not add empty lines that are not needed

suryaanshah replied:
deleted the extra line now in commit: [8b705e3](https://github.com/freeCodeCamp/freeCodeCamp/pull/56611/commits/8b705e3e566119548c0f2c6937818a1fd3d0c447)

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
remove empty line

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,614 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder answer
+```css
+a[href^="http"] { 
````
Why is this bad?

Feels like a subjective thing.

suryaanshah replied:
The idea is that it will highlight all links on the page. Compared to other options this seems unnecessary. Also, if a developer does want a formatting in all the links, they can do so in simpler ways. 

Although on a second thought, this can have a use-case since all links may not be http.

I'll change this question.

suryaanshah replied:
Hello, I have replaced this question with a new one.

suryaanshah committed:
replaced q1

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
 
-Placeholder question
+ How would you target all `<input>` elements with an id attribute that ends with `"field"`?
```
```suggestion
How would you target all `<input>` elements with an `id` attribute that ends with `"field"`?
```

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
 
-Placeholder question
+How would you style `<a>` elements where the title attribute starts with `"Read"`?
```
```suggestion
How would you style `<a>` elements where the `title` attribute starts with `"Read"`?
```

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder answer
+```css
+button:not([data-tooltip="off"]) { text-decoration: underline dotted; }
````
Would this not target button elements that have no `data-tooltip` at all?

suryaanshah replied:
fixed this issue.
Thank you for pointing it out!

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Naomi <accounts+github@nhcarrigan.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Naomi <accounts+github@nhcarrigan.com>

suryaanshah committed:
fix q10

fixed the answer of q10 as it was including all `button` tags without the `data-tooltip` attribute. Thanks to @naomi-lgbt for pointing this out.

suryaanshah commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -145,7 +145,7 @@ a[title^="Read"] { color: blue; }
 #### --text--
 
- How would you target all `<input>` elements with an id attribute that ends with `"field"`?
+How would you target all `<input>` elements with an `id` attribute that ends with `"field"`?
```
fair enough. I have used the `///` wherever I wrote a code-related term.

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Ilenia <26656284+ilenia-magoni@users.noreply.github.com>

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
 
-Placeholder question
+Which of the following will not run as expected with this style?
```
```suggestion
Which of the following will not match this CSS rule?
```

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
 
-Placeholder question
+You need to style all links (`a`) that point to image files by selecting href attributes ending with `.jpg`, `.png`, or `.gif`. Which selector works best?
```
```suggestion
You need to style all links (`a`) that point to image files by selecting `href` attributes ending with `.jpg`, `.png`, or `.gif`. Which selector works best?
```

naomi-lgbt commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
```diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
 #### --text--
 
-Placeholder question
+You want to style ordered lists (`ol`) where the type attribute is set to `"i"` (for Roman numerals). Which of the following selectors would you use?
```
```suggestion
You want to style ordered lists (`ol`) where the `type` attribute is set to `"i"` (for Roman numerals). Which of the following selectors would you use?
```

naomi-lgbt reviewed (changes requested):
Thanks for all your work on this. Last one, I think.

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Naomi <accounts+github@nhcarrigan.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Naomi <accounts+github@nhcarrigan.com>

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Naomi <accounts+github@nhcarrigan.com>

gikf commented on curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md:
````diff
@@ -17,439 +17,606 @@ Answer all of the questions below correctly to pass the quiz.
-
+```css
+div[data-status$="open"] { border: 2px solid green; }
+```
````
````suggestion
```

````
Missing blank line causes linting to fail.

suryaanshah committed:
Update curriculum/challenges/english/25-front-end-development/quiz-css-attribute-selectors/66ed8fd0f45ce3ece4053eaf.md

Co-authored-by: Krzysztof G. <60067306+gikf@users.noreply.github.com>

gikf reviewed (approved):
Note for future - it is recommended to make changes on a separate branch, rather than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/how-to-contribute-to-the-codebase/#contributing-to-the-codebase.

gikf commented:
Hi @suryaanshah. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝
